bin/GuiConfigs/Nekotekina by GooseWing.qss

```diff
@@ -273,9 +269,18 @@ QSlider#sizeSlider::groove:horizontal {
 	height: 1.5em;
 }
 
+

```

Comment:
Sussy newline

Reply:
Does it hurt your eyes?

Reply:
Yes

```suggestion
```

Reply:
In case you didn't notice: the stylesheets always had 2 newlines between sections.
But whatever